soh/src/overlays/actors/ovl_En_GirlA/z_en_girla.c

```diff
@@ -861,6 +869,11 @@ void EnGirlA_BuyEvent_ZoraTunic(GlobalContext* globalCtx, EnGirlA* this) {
 }
 
 void EnGirlA_BuyEvent_ObtainBombchuPack(GlobalContext* globalCtx, EnGirlA* this) {
+    Rupees_ChangeBy(-this->basePrice);
+
+    if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_BOMBCHUS_IN_LOGIC)) {
+        return;
+    }
```

Comment:
could you add a comment explaining what's happening here? it's not clear why this early return exists